site/app/templates/grading/simple/Display.twig

```diff
@@ -1,20 +1,8 @@
 {% import _self as self %}
 
-<div class="content">
-    {#
-        Default is viewing your sections
-        Limited grader does not have "View All" option
-        If nothing to grade, Instructor will see all sections
-    #}
-    {% if show_all_sections_button %}
-        <div style="float: right; margin-bottom: 10px">
-            <a class="btn btn-default"
-               href="{{ core.buildUrl({'component': 'grading', 'page': 'simple', 'action': action, 'g_id': gradeable.getId(), 'sort': sort, 'view': view_all ? null : "all"}) }}">
-                {{ view_all ? "View Your Sections" : "View All" }}
-            </a>
-        </div>
-    {% endif %}
+<link rel="stylesheet" type="text/css" href="{{ base_url }}css/simple-grading.css" />
```

Comment:
I probably should have mentioned this before, but there's a function that we call in PHP called `AddCssInternal` that we use to attach CSS (and another one for JS files) so we can force users to update their CSS/JS cache when the files change.  If you find the place where `Display.twig` is being rendered in PHP you can add a call to `AddCSSInternal` there.

Reply:
Fixed